src/app/src/utils/canvas.js

```diff
@@ -6,12 +6,80 @@ export const combineCanvases = (canvasList, direction) => {
     }
 };
 
+function generateWrappedText(context, text, width, font) {
+    context.font = font;
+
+    var x = 20;
+    var y = 50;
+    var maxWidth = width - x * 2;
+    var lineHeight = 50;
+
+    var words = text.split(' ');
+    var line = '';
+    const lines = [];
+    for (var n = 0; n < words.length; n++) {
```

Comment:
A raw for loop in the wild. 🦓

Reply:
😂 In my first pass at this, I switched to `forEach` and it broke! In my current implementation it seems like `forEach` is more cooperative, so I've switched it over.

Reply:
That always happens to me in PR reviews haha